model/src/main/java/org/cbioportal/genome_nexus/model/my_variant_info_model/Alleles.java

```diff
@@ -0,0 +1,21 @@
+package org.cbioportal.genome_nexus.model.my_variant_info_model;
+
+import org.springframework.data.mongodb.core.mapping.Field;
+
+public class Alleles
```

Comment:
Seems like this class, and all other model classes under `my_variant_info_model` package, have indentation by `tab` character instead of `space`. For consistency, I suggest replacing tabs with 4 space characters.